test/acceptance/tests/controller/controller_namespaces_test.go

```diff
@@ -173,6 +191,9 @@ func TestControllerNamespaces(t *testing.T) {
 				t.Log("deleting service-resolver CRD")
 				helpers.RunKubectl(t, ctx.KubectlOptions(t), "delete", "-n", KubeNS, "serviceresolver", "resolver")
 
+				t.Log("deleting service-router CRD")
```

Comment:
Should we use `CRs` here too instead of `CRDs` for consistency? Not a huge deal either way for me.

Reply:
Yea for sure, I'll make a note and we'll do this after service-router and proxy-defaults land so we get both.

Reply:
Actually the other PRs landed so I got this in here!